src/efr32mg1/README.md

```diff
@@ -233,7 +199,7 @@ The above example demonstrates basic OpenThread capabilities. Enable more featur
 ```bash
 $ cd <path-to-ot-efr32>
 $ ./bootstrap
```

Comment:
Shouldn't this be ./script/bootstrap?

Reply:
Is `bootstrap` even required? Can we remove this line?